pkg/util/cluster/cluster.go

```diff
@@ -368,6 +388,54 @@ func (c *Cluster) createCluster(ctx context.Context, clusterName, clientID, clie
 	return c.openshiftclusters.CreateOrUpdateAndWait(ctx, c.ResourceGroup(), clusterName, oc)
 }
 
+func (c *Cluster) createClusterPreview(ctx context.Context, clusterName, clientID, clientSecret string) error {
```

Comment:
investigate to see if we can define the cluster in the internal representation and use api converters to generate the correct api manifest for the version of client that's being used